lib/rubocop/cop/mixin/range_help.rb

```diff
@@ -34,6 +34,21 @@ def contents_range(node)
         range_between(node.loc.begin.end_pos, node.loc.end.begin_pos)
       end
 
+      # A range containing the first to the last argument
+      # of a method call or method definition.
+      # def foo(a, b:)
+      #         ^^^^^
+      # bar(1, 2, 3, &blk)
+      #     ^^^^^^^^^^^^^
+      # baz { |x, y:, z: }
```

Comment:
Looks like you're missing a pipe here.
```suggestion
      # baz { |x, y:, z:| }
```